backend/forms/forms.py

```diff
@@ -2,7 +2,45 @@
 from wtforms import (StringField, IntegerField, DateField,
                      HiddenField, PasswordField, SelectField,
                      SubmitField)
-from wtforms.validators import DataRequired, Email, Optional, Length, Regexp
+from wtforms.validators import (DataRequired, Email, Optional,
+                                Length, Regexp, ValidationError)
+
+from app import db
+from models.users import User
+
+
+class Check_users_field(object):
```

Comment:
Bad style for class name. If you take a look to WTForm's buit-in validators, you will see that all of them use CamelCase. Anyway, I'd recommend something like `UniqueValue` name, as it better reflects the purpose of this validator.